libs/api_common/windows_platform_common.cpp

```diff
@@ -276,6 +283,8 @@ _get_section_definition(const std::string& section)
 _Ret_maybenull_ const ebpf_program_type_t*
 get_ebpf_program_type(bpf_prog_type_t bpf_program_type)
 {
+    _load_ebpf_provider_data();
```

Comment:
nit: Since we have call to this function duplicated in multiple places, is it possible to wrap the global maps in a getter function and the getter function internally calls `_load_ebpf_provider_data()`?

Reply:
This is only called from the top-level non-static APIs to avoid calling it unnecessarily.